src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -543,6 +552,7 @@ private void setupDrivetrainShuffleboard() {
     if (!Constants.kUseTelemetry) return;
 
     m_drivetrainTab.add("Field", m_fieldDisplay);
+    SmartDashboard.putData("Field Display", m_fieldDisplay);
```

Comment:
We shouldn't need this if the field is already being put onto the drivetrain tab. You can view the field by using the field on the drivetrain tab, both in the simgui and in shuffleboard.